tests/test_review_dc_people.py

```diff
@@ -0,0 +1,86 @@
+
+In each of the court cases, Amos revealed his apostate sentiments. These sentiments were confirmed in a June 20, 1844 Robert D. Foster letter written in Carthage, Illinois: “Tell Amos Davis to keep his eyes open, as we learn that consecration law will soon commence on him. This we know, and he had better look out sharp.”[6](#f6)
+
+Amos did not join the Latter-day Saint exodus from Nauvoo to the Territory of Iowa. In 1847 he sued Church trustees for debts owed him. In the late 1840s, Amos journeyed to the west. Nancy Tracy wrote, “In the early part of winter, my husband’s old employer, Amos Davis, came along through our settlement with a load of goods from his store in Nauvoo. He stopped with us for a day. When he left, he gave us some tea, sugar, and coffee, which was highly appreciated and was a luxury in those hard times.”[7](#f7)
+
+By 1850 Amos was residing in Northern California. By 1853 he was living in Michigan. Five years later, he had returned to Illinois. Amos died on March 22, 1872 near Nauvoo at age fifty-eight.
+
+[1](#t1). Nancy Tracy Autobiography, typescript, p. 27. L. Tom Perry Special Collections, Harold B. Lee Library, Brigham Young University, Provo, UT.
+
+[2](#t2). Journal, December 1841–December 1842, p. 89. Joseph Smith Papers.
+
+[3](#t3). Docket Entry, 30 November 1842 [*City of Nauvoo v. Davis for Slander of JS–B*], p. 10. Joseph Smith Papers.
+
+[4](#t4). Smith, *History of the Church*, 5:198.
+
+[5](#t5). Docket Entry, between circa 23 and circa 24 November 1843 [*State of Illinois v. Finch*], p. 89. Joseph Smith Papers.
+
+[6](#t6). History, 1838–1856, volume F-1 [1 May 1844–8 August 1844], p. 130. Joseph Smith Papers.
+
+[7](#t7). Nancy Tracy Autobiography, p. 35.
+
+"""
+
+
+def check_text(markdown_content: str) -> Any:
+    """Check for unwanted text presence"""
+    # result check variable boolean
+    result_check = []
+    markdown_content = clean_text(markdown_content)
+
+    # Search for the position of "abstract" (case insensitive)
+    text_match = re.search(r"\b[\d+](#t\d+).\b", markdown_content, re.IGNORECASE)
+
+    if text_match:
+        # Check content after the "text" text
+        text_position = text_match.start()
+
+        # Check if there is text after the "text" section
+        if text_position > 0:
+            result_check.append("True")
+        else:
+            result_check.append("False")
+
+    return result_check
+
+
+def test_review_load_dc_people() -> None:
+    """It returns a valid Document for load dc people."""
+    result = check_text(mark_text)
```

Comment:
your test should pass in HTML to the load_dc_people function and verify that the page_content no longer has footnotes.